pkg/pod/entrypoint.go

```diff
@@ -135,12 +135,20 @@ func orderContainers(commonExtraEntrypointArgs []string, steps []corev1.Containe
 		)
 		argsForEntrypoint = append(argsForEntrypoint, commonExtraEntrypointArgs...)
 		if taskSpec != nil {
-			if taskSpec.Steps != nil && len(taskSpec.Steps) >= i+1 {
-				if taskSpec.Steps[i].Timeout != nil {
-					argsForEntrypoint = append(argsForEntrypoint, "-timeout", taskSpec.Steps[i].Timeout.Duration.String())
+			if taskSpec.Steps != nil {
+				if len(taskSpec.Steps) >= i+1 {
+					if taskSpec.Steps[i].OnError != "" {
+						argsForEntrypoint = append(argsForEntrypoint, "-on_error", taskSpec.Steps[i].OnError)
+					}
+					if taskSpec.Steps[i].Timeout != nil {
+						argsForEntrypoint = append(argsForEntrypoint, "-timeout", taskSpec.Steps[i].Timeout.Duration.String())
+					}
+				}
+				if taskSpec.Steps[i].StdoutConfig != nil {
+					argsForEntrypoint = append(argsForEntrypoint, "-stdout_path", taskSpec.Steps[i].StdoutConfig.Path)
 				}
-				if taskSpec.Steps[i].OnError != "" {
-					argsForEntrypoint = append(argsForEntrypoint, "-on_error", taskSpec.Steps[i].OnError)
+				if taskSpec.Steps[i].StderrConfig != nil {
+					argsForEntrypoint = append(argsForEntrypoint, "-stderr_path", taskSpec.Steps[i].StderrConfig.Path)
 				}
```

Comment:
Sorry, I think this was bad advice on my side, `len(taskSpec.Steps) >= i+1` actually includes all steps 🙏